docs/quick-start.md

```diff
@@ -33,18 +33,21 @@ After everything is installed, you're ready to run Jetpack locally! While there
 
 To setup Docker:
 - Install Docker:
-	- Mac: `brew install --cask docker`
+	- Mac: `brew install --cask docker` (This will take a while!)
 	- Linux: `brew install docker` 
-- Copy the settings file and modify the site password and any other configuration you'd like: `cp tools/docker/default.env tools/docker/.env`
-- Start the Docker container using `jetpack docker up`
+	- `open -a Docker` (or open the app from your Applications folder) to open the GUI application. You will likely need to enter your device password and accept their terms for a first time setup.
```

Comment:
This only applies to Mac.

Reply:
Would it be better to just say "Open the Docker application"?

Reply:
Maybe "Open the Docker application or run `open -a Docker` (mac)"

Reply:
I think we should make sure that folks have a license for Docker Desktop now also. A pointer at the field guide?

Reply:
There's not really a "Docker application" for Linux. You just run things in the CLI.

I don't know whether `brew install docker` starts the docker daemon though. Hopefully it does.

Reply:
> I think we should make sure that folks have a license for Docker Desktop now also. A pointer at the field guide?

Doesn't Docker Desktop pop up something about that? Let that handle it IMO, no need to clutter our docs about it.

Reply:
Agreed re licensing, at least for this quick guide doc and for the v1. We can mention it elsewhere.

Reply:
Re what to do on Linux, I'm setting up some VMs to test this on clean machines and will specifically check the Linux step here.

Maybe update line 36 to:
- Mac: `brew install --cask docker && open -a Docker`?